fe/lib/components/Questionnaire/QuestionnaireBuilder/FormBuilder/state/formBuilderState.ts

```diff
@@ -140,12 +151,16 @@ export const actionCreators = {
     PrivacyConsent: createEditPrivacyConsentAction,
   },
   delete: createDeleteFreeTextAction,
+  import: importComponentsAction,
 } as const;
 
 interface StateContextType {
   state: FormComponent[];
   dispatch: React.Dispatch<
-    AddComponentAction | DeleteComponentAction | EditComponentAction
+    | AddComponentAction
+    | DeleteComponentAction
+    | EditComponentAction
+    | ImportComponentAction
```

Comment:
I think this should be plural, like `ImportComponents/ReplaceComponents/SetComponents`